### ISSUE TITLE
RCS1097 False Negative 2

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#347]
The following code doesn't throw RCS1097 Remove redundant 'ToString' call.

```csharp
int x = 0;
Console.WriteLine("Test" + x.ToString());
```

Isn't this a false negative?

**Comment by JosefPihrt** [comment]
This one is not false negative because removal of 'ToString' would cause boxing.
